src/components/gltf-model.js

```diff
@@ -22,7 +22,7 @@ module.exports.Component = registerComponent('gltf-model', {
     this.remove();
 
     this.loader.load(src, function gltfLoaded (gltfModel) {
-      self.model = gltfModel.scene;
+      self.model = gltfModel.scene || gltfModel.scenes[0];
```

Comment:
`gltfModel.scene` can be one of `gltfModel.scenes[0~gltfModel.scenes.length-1]`,
or `undefined` if default scene isn't specified.

default scene indicates the index in `gltfModel.scenes`.
`gltfModell.scene` = `gltfModel.scenes[defaultScene];`.
But default scene is optional in glTF specification.
If default scene isn't specified in the glTF data file,
`gltfModel.scene` will be `undefined`.

So `gltfModel.scene` can be `undefined` even though gltfModex.scenes[0] exists.